slurp.1.scd

```diff
@@ -44,6 +44,12 @@ held, the selection is moved instead of being resized.
 	Set color for highlighting predefined rectangles from standard input when not
 	selected.
 
+*-F* _font family_
+	Set the font family name when displaying the dimensions box. Only useful
+	when combined with the -d option. The available font family names are the
+	standard generic CSS2 options: serif, monospace or sans.
```

Comment:
The generic CSS2 options also include `fantasy` and `cursive`, and it is `sans-serif` not `sans` (see https://www.w3.org/TR/CSS2/fonts.html#generic-font-families). Other fonts families may also work, but there isn't any guarantee. I'm not sure if that is worth mentioning.

Reply:
Right, I've updated my patch to mention about the other generic font family names and replaced to use `sans-serif` instead of `sans` (I just kept like that because it was being used on the original implementation).